sbt-plugin/src/sbt-test/gen-scala-server/01-gen-basic-server/build.sbt

```diff
@@ -2,7 +2,9 @@
 // https://github.com/akka/akka-grpc/pull/1279
 scalaVersion := "2.12.18"
 
+// TODO remove these resolvers when we start using released Pekko jars
 resolvers += Resolver.ApacheMavenSnapshotsRepo
+resolvers += "apache-staging".at("https://repository.apache.org/content/groups/staging/")
```

Comment:
This would require an upstream contribution to sbt, not against it but there are pros and cons to this. Our usage of staging here is temporary, once the first release of Pekko is made it will never be in the codebase again.

Reply:
I see! thanks.